src/components/SpeakersSection/allspeakers.jsx

```diff
@@ -0,0 +1,7 @@
+export const IMAGE_LINKS = [
```

Comment:
This should probably be export default, and it should be more data structure like, but this is all I really asked for, so no worries.